AdvancedDLSupport.Tests/c/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ list(
         SymbolTransformationTests
         GenericDelegateTests
         BooleanMarshallingTests
+        ReturnsSpanTests
```

Comment:
You need to update these as well for unit tests to work. The project & test file name should match what you specify as the library name in the test class.